torch/_torch_docs.py

```diff
@@ -1500,6 +1500,84 @@ def merge_dicts(*dicts):
+is calculated, which is given by
+
+.. math::
+    \text{cov}_w(x,y) = \frac{\sum^{N}_{i = 1}w_i(x_{i} - \mu_x^*)(y_{i} - \mu_y^*)}{\sum^{N}_{i = 1}w_i~-~1}
+
+where :math:`\mu_x^* = = \frac{\sum^{N}_{i = 1}w_ix_{i} }{\sum^{N}_{i = 1}w_i}` 
+is the weighted mean of the variable. 
+
+
+Args:
+    input (Tensor): A 1-D or 2-D tensor containing multiple variables and observations.
+        Each row of :attr:`input` represents a variable, and each column a
+        single observation of all those variables. Also see :attr:`rowvar` below.
+    other (Tensor, optional): An additional set of variables and observations.
+        :attr:`other` has the same form as that of :attr:`input` and can be used to provide
+        additional variables for each of the observations which would be considered along with those provided in
+        :attr:`input`. 
+    rowvar (bool, optional): If :attr:`rowvar` is `True` (default), then each row represents a variable,
+        with observations in the columns.
+        Otherwise, the relationship is transposed: each column represents a variable,
+        while the rows contain observations.
+    bias (bool, optional): Default normalization (`False`) is by :math:`(N - 1)` which produces the unbiased covariance estimate,
+        where :math:`N` is the number of observations given.
+        If :attr:`bias` is `True`, then normalization is by :math:`N`.
+        These values can be overridden by using the keyword :attr:`ddof`.
+    ddof (int, optional): If not `None` the default value implied by :attr:`bias` is overridden.
+        Note that :attr:`ddof`:math:`=1` will return the unbiased estimate,
+        even if both :attr:`fweights` and :attr:`aweights` are specified,
+        and :math:`ddof`:math:`=0` will return the simple average.
+    fweights (tensor, optional): 1-D tensor of integer frequency weights;
+        the number of times each observation vector should be repeated.
+    aweights (tensor, optional): 1-D array of observation vector weights.
+        These relative weights are typically large for observations considered “important” and smaller for 
+        observations considered less “important”. If :attr:`ddof`:math:`=0` the tensor of weights can be used to
+        assign probabilities to observation vectors.
+
+Example::
+    >>> x =  torch.tensor([[0, 2], [1, 1], [2, 0]]).T
+    >>> x
+    tensor([[0, 1, 2],
+            [2, 1, 0]])
+    >>> torch.cov(x)
+    tensor([[ 1., -1.],
+            [-1.,  1.]], dtype=torch.float64)
+    >>> x = torch.tensor([-2.1, -1,  4.3])
+    >>> y = torch.tensor([3,  1.1,  0.12])
+    >>> X = torch.stack ((x, y), 0)
+    >>> torch.cov(X)
+    tensor([[11.7100, -4.2860],
+            [-4.2860,  2.1441]])
+    >>> torch.cov(x, y)
+    tensor([[11.7100, -4.2860],
+            [-4.2860,  2.1441]])
+    >>> torch.cov(x)
+    tensor(11.7100)
+    >>> x = torch.rand(3,10)
+    >>> f = torch.arange(10) * 2
```

Comment:
This is a fine example that shows how to clal the operation with fweights and aweights, but what about using fweights and aweights with the original matrix that has just a few elements, and talking about how setting different values is changing the result?

I really like, for example, how you show that using input and other is equivalent to stacking. Maybe a comment could be added to that part of the example, too, highlighting for users what you're showing them?

Doing the same for fweights and aweights would be really helpful.